src/components/screens/IntegrationsCatalog/RecipesDetailScreen/RecipesDetailScreen.js

```diff
@@ -218,7 +218,7 @@ export const RecipesDetailScreen = ({ path, location, pageContext }) => {
               {breadcrumb.title}
             </SubNavBreadcrumb>
             <SubNavRight>
-              <SubNavCTA href="/docs/react/addons/addon-catalog/">
+              <SubNavCTA href="/docs/react/integrations/integration-catalog/">
```

Comment:
Should this also be updated? https://github.com/storybookjs/frontpage/blob/main/src/components/screens/IntegrationsCatalog/AddonsDetailScreen/AddonsDetailScreen.js#L195

Reply:
good catch KG